src/ctree-app/features-section.html

```diff
@@ -0,0 +1,120 @@
+<link rel="import" href="../../bower_components/polymer/polymer-element.html">
+<link rel="import" href="../../bower_components/polymer/lib/dom-repeat.html">
+<link rel="import" href="../ctree-icons/ctree-landing-icons.html">
+
+<dom-module id="features-section">
+  <template>
+    <style>
+      :host {
+        display: block;
+        background-color: #FFFFFF;
+        color: #196848;
+        padding-top: var(--app-header-padding-fix);
+        margin-top: var(--app-header-margin-fix);
+      }
+      iron-icon {
+        --iron-icon-fill-color: #62A374;
+        --iron-icon-width: 80px;
+        --iron-icon-height: 80px;
+      }
+      .review-icon {
+        transform: rotate(40deg);
+      }
+      h1, h3 {
+        color: #62A374;
+      }
+      h1 {
+        text-align: center;
+        font-size: 2.9em;
+        letter-spacing: .02em;
+      }
+      h3 {        
+        text-transform: capitalize;
+        margin-top: 0;
+      }
+      p {
+        color: #222322;
+        font-size: 0.75em;
+        letter-spacing: .027em;
+        padding-top: 10px;
+      }
+      .container {
+        display: flex;
+        flex-wrap: wrap;
+        justify-content: space-evenly;
+        max-width: 1000px;
+        margin: auto;
+      }      
+      .collaboration, .search, .share, .review {
+        display: flex;
+        width: 350px;
+        margin-bottom: 50px;
+      }
+      .image {
+        margin-right: 50px;
+      }
+      /* Mobile Phones */
+      @media (max-width: 500px) {
+        h3 {
+          font-size: 1.25em;
+        }
+        div p {
+          font-size: 1.12em;
+          margin: 0 45px;
+        }
+        div .collaboration, div .search, div .share, div .review {
+          display: block;
+          width: 90%;
+        }
+        div .image {
+          margin-right: 0;
+        }
+      }
+    </style>
+
+    <h1>Features</h1>
+    <div class="container">
+      <dom-repeat items="[[sections]]" as="section">
+        <template>
+          <div class$="[[section.name]]">
+            <div class="image">
+              <iron-icon class$="[[section.name]]-icon" icon="[[section.name]]"></iron-icon>
+            </div>
+            <div>
+              <h3>[[section.name]]</h3>
+              <p>
+                [[section.description]]
+              </p>
+            </div>
+          </div>
+        </template>
+      </dom-repeat>
+    </div>
+  </template>
+
+  <script>
+  
+    class FeaturesSection extends Polymer.Element {
+      static get is() { return 'features-section'; }
```

Comment:
I think our standard has been to add some empty lines between functions and things.  We need a style guide, but in the absence for that it's best to try to follow the style from existing components.  If you feel strongly that this is a better practice or more in line with the industry I'm happy to reconsider this and set a different standard we can move toward, but given the volume of existing code I'd need some convincing.

Reply:
Oh no, I can definitely agree to add some empty lines between functions and things. I don't feel strongly about my way at all; it was just how utilziing `poly init` creates its style.

New commit has spacing between functions and things.